modal/cli/entry_point.py

```diff
@@ -20,6 +29,15 @@
     """,
 )
 
+
+@entrypoint_cli.callback()
+def root(
+    ctx: typer.Context,
+    version: bool = typer.Option(None, "--version", callback=version_callback),
```

Comment:
wow, hacky!

Reply:
haha yeah... would be nice if there's some slightly less hacky way of doing it. but not a big deal.